src/current/_includes/releases/cloud/2023-06-05.md

```diff
@@ -0,0 +1,19 @@
+## June 05, 2023
+
+<h3> Console changes </h3>
+
+- Organizations that have purchased premium support will now see it included in their [invoices](../cockroachcloud/billing-management.html).
+- Cross-cluster [restores](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) are now limited to {{ site.data.products.dedicated }} clusters with a major version greater than or equal to the major version of the source cluster.
+- The [**Restore jobs**](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) tab of the **Backups page** now shows more information about the restore job, such as the source and destination clusters, the restore type, the backup size, and the job progress.
+
+<h3> Security updates </h3>
+
+- The [Organization Audit Logs API](../cockroachcloud/cloud-org-audit-logs.html), which provides logs of events that occur in a Cloud organization, is now [generally available](../{{site.versions["stable"]}}/cockroachdb-feature-availability.html#feature-availability-phases).
+- `ExternalId` support is now re-enabled in the [Customer-managed Encryption Key (CMEK)](../cockroachcloud/cmek-ops-aws.html) capability on AWS.
+
+<h3> Bug fixes </h3>
+
+- The `status` returned by the [`logexport` Cloud API endpoint](../cockroachcloud/export-logs.html#the-logexport-endpoint) is now determined by the readiness of the underlying logging resources, in addition to the latest log export update job state. Before this change, it was possible for `GET` requests to the `logexport` to report a log export status that conflicted with the latest log export update job state or with the state of the logging infrastructure.
+- Fixed a bug where concurrent [restores](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) could run on the same destination cluster.
+- Fixed a bug where the IOPS price preview shown when [creating](../cockroachcloud/create-your-cluster.html) or [editing a cluster](../cockroachcloud/cluster-management.html) was inaccurate.
+- The **Group** tab is now only shown to users who have access to it.
```

Comment:
```suggestion
- The **Group** tab now appears only to users who have the `TBD` privilege.
```
Is this based on a specific privilege? I left it as TBD in my suggestion, but you should be able to edit it. Was this a privilege escalation bug that has been fixed, where people could see info they shouldn't have access to, or would the Group tab appear before but be empty if you didn't have access?

Reply:
the page was empty if you didn't have access, i'll follow up with eng about who has access to the page. seems like it was supposed to be behind a feature flag.